packages/tempus/test/Subcycling/Tempus_SubcyclingTest.cpp

```diff
@@ -410,6 +417,10 @@ TEUCHOS_UNIT_TEST(Subcycling, VanDerPolOperatorSplit)
     stepperSC->setSubcyclingMaxFailures      (10);
     stepperSC->setSubcyclingMaxConsecFailures(5);
     stepperSC->setSubcyclingScreenOutputIndexInterval(1);
+    //stepper->setSubcyclingIntegratorObserver(
+    //  Teuchos::rcp(new Tempus::IntegratorObserverSubcycling<double>()));
+    //stepperSC->setSubcyclingPrintDtChanges   (true);
+
```

Comment:
Needed?

Reply:
I kept these for when we want to output the subcycling timestep information during debugging.